readme.md

```diff
@@ -32,35 +32,35 @@ Once installation is working, try creating and requesting resources. Note that t
 
 ### Instructions & Guiding Questions
 
-- [ ] Take a moment to look through the code that already exists in this repository. Run the code and test out each route, ensuring it does what you expect.
+- [x] Take a moment to look through the code that already exists in this repository. Run the code and test out each route, ensuring it does what you expect.
 
 ---
 
-- [ ] We will be using both [mongoose](https://mongoosejs.com/docs/guide.html) and [mongodb](https://docs.mongodb.com/manual/) to build this project. Open up each project's documentation.
+- [x] We will be using both [mongoose](https://mongoosejs.com/docs/guide.html) and [mongodb](https://docs.mongodb.com/manual/) to build this project. Open up each project's documentation.
 
 * **Question:** What is the difference between mongoose and mongodb?
 
 * **Your Answer:**
-
+MongoDB is a non-relational database; mongoose is a javascript library for working with Mongo.
 ---
 
-- [ ] MongoDB uses a number of terms that may be to new to you such as database, collection, and document.
+- [x] MongoDB uses a number of terms that may be to new to you such as database, collection, and document.
 
 * **Question:** What is the difference between the above terms? [This page](https://docs.mongodb.com/manual/core/databases-and-collections/) may help.
 
 * **Your Answer:**
-
+A document is a single key-value pair in a collection. A collection is a record within a database.
```

Comment:
This is not quite right. Databases can have multiple collections. Collections have multiple documents. Documents can have multiple key-value pairs inside of them which represent a single object.